variables.tf

```diff
@@ -1,5 +1,11 @@
 variable "domain" {
   description = "DNS Domain to update"
   type        = string
-  default     = "statusim.net"
+  default     = "status.im"
+}
+
+variable "cf_zone_id" {
+  description = "ID of CloudFlare zone for host record."
+  type        = string
+  default     = "fd48f427e99bbe1b52105351260690d1" /* status.im */
```

Comment:
Why tho? Can't you just get it from `local.zones["status.im"]`? That's why we have this:
https://github.com/status-im/infra-misc/blob/baa8b5c6f915f9d8fdf620dc574fec37d35f1c56/main.tf#L16-L29

Reply:
Removed